src/test/java/io/xstefank/wildfly/bot/PRFormatOverrideMessage.java

```diff
@@ -95,4 +95,76 @@ public void testOverridingCommitMessage() throws IOException {
                             .formatted(String.format("- For commit: \"%s\" (%s) - Custom commit message", commitMessage, gitHubJson.commitSHA())));
                 });
     }
+
+    @Test
+    public void testOverridingTitleEnabledToTrue() throws IOException {
```

Comment:
This can use the JSON Builder for all the constants. I'd like to believe the Builder was implemented during the creation of this PR. 

It would also benefit from using the Utils method once #118 is merged. New place for the test should be PRCommitCheckTest

Reply:
The builder is not currently in any PR. I am working on different issues and I have created the builder there. Soon I will open PR with it. It will however add new features, so it might get delayed due to that.